### DISCUSSION TITLE
Queue API

### DISCUSSION THREAD

**Discussion by AIjie-sen** [Comfy-Org/ComfyUI#2073] [Q&A]
How to query the queue status, add queues, and eliminate queues in comfyUI through API

**Comment by JokerMartini** [comment]
did you ever figure anything out?

**Comment by ltdrdata** [comment]
queue status
https://github.com/comfyanonymous/ComfyUI/blob/8508df25691b0c9213049ab0d723610d3d8f9136/web/scripts/api.js#L246

add queue
https://github.com/comfyanonymous/ComfyUI/blob/8508df25691b0c9213049ab0d723610d3d8f9136/web/scripts/api.js#L211

delete
https://github.com/comfyanonymous/ComfyUI/blob/8508df25691b0c9213049ab0d723610d3d8f9136/web/scripts/api.js#L290
https://github.com/comfyanonymous/ComfyUI/blob/8508df25691b0c9213049ab0d723610d3d8f9136/web/scripts/api.js#L310

**Comment by zhzLuke96** [comment]
If you use nodejs, i recommend checking out my (js) client implementation, which includes the complete comfyui API

https://github.com/StableCanvas/comfyui-client

and document here:
https://stablecanvas.github.io/comfyui-client/classes/ComfyUIApiClient.html
